PhysicsTools/PatAlgos/python/slimming/metFilterPaths_cff.py

```diff
@@ -55,3 +55,27 @@ def miniAOD_customizeMETFiltersFastSim(process):
 phase2_hgcal.toReplaceWith( Flag_HBHENoiseIsoFilter, cms.Path() )
 phase2_hgcal.toReplaceWith( Flag_eeBadScFilter, cms.Path() )
 
+metFilterPathsTask = cms.Task(
+    HBHENoiseFilterResultProducer,
+    HBHENoiseFilter,
```

Comment:
@mmarionncern @zdemirag I could make this question a little more specific. When metFilterPaths_cff is loaded do we want these filters to be setup by default to run unscheduled? If they are on an explicit Path there is no reason to do this. If we do not want them to run at all, there is no reason to do this. The only thing that loads this file is PhysicsTools/PatAlgos/python/slimming/slimming_cff.py and that file is most often loaded by ConfigBuilder when PAT is included. There are also a small number of test configuration files that explicitly load slimming_cff. Even if we delete the Task from metFiltersPaths_cff, one could still write a test configuration that defines such a Task for some special purpose. The question is do we want these filters to be setup by default for unscheduled whenever PAT is included by ConfigBuilder, and this setup implies the filter modules would be constructed by the C++ part of the Framework.